tests/SprykerSdkTest/Zed/AppSdk/Business/AppManifestFacadeTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Zed\AppSdk\Business;
+
+use Codeception\Test\Unit;
+
+/**
+ * @group SprykerSdkTest
+ * @group Zed
+ * @group AppSdk
+ * @group Business
+ * @group AppManifestFacadeTest
+ */
+class AppManifestFacadeTest extends Unit
+{
+    /**
+     * @var string
+     */
+    public const INVALID_LOCALE = 'en_U';
+
+    /**
+     * @var \SprykerSdkTest\Zed\AppSdk\BusinesssTester
+     */
+    protected $tester;
+
+    /**
+     * @return void
+     */
+    public function testAddManifestAddsANewManifestFile(): void
+    {
+        // Arrange
+        $manifestRequestTransfer = $this->tester->haveManifestCreateRequest();
+
+        // Act
+        $manifestResponseTransfer = $this->tester->getFacade()->createManifest(
+            $manifestRequestTransfer,
+        );
+
+        // Assert
+        $this->tester->assertManifestResponseHasNoErrors($manifestResponseTransfer);
+        $this->assertFileExists($manifestRequestTransfer->getManifestPath());
+    }
+
+    /**
+     * @return void
+     */
+    public function testAddManifestWithInvalidLocaleReturnsErrorResponse(): void
+    {
+        // Arrange
+        $manifestRequestTransfer = $this->tester->haveManifestCreateRequest();
+        $manifestRequestTransfer->getManifestOrFail()->setLocaleName(static::INVALID_LOCALE);
+
+        // // Act
+        $manifestResponseTransfer = $this->tester->getFacade()->createManifest(
+            $manifestRequestTransfer,
+        );
+
+        // Assert
+        $this->assertSame(
+            'en_U',
```

Comment:
You don't test anymore with this. 

$manifestRequestTransfer->getManifestOrFail()->setLocaleName(static::INVALID_LOCALE);
and 
$manifestRequestTransfer->getManifestOrFail()->getLocaleName()

Will always be the same.

Use `assertCount(1, $manifestResponseTransfer->getErrors())` or write a specific assertio for your case in the Helper.

`$this->tester->assertInvalidLocaleErrorResponse($manifestResponseTransfer)`

```
public function assertInvalidLocaleErrorResponse(ManifestResponseTransfer $manifestResponseTransfer) {
     $errorTransfers = $manifestResponseTransfer->getErrors();

     $this->assertCount(1, $errorTransfers, sprintf('Expected to get exactly "1" error, got "%s" errors. Either there is no error or you have more than expected',
                $manifestResponseTransfer->getErrors()->count());

     $localeErrorTransfer = ''; // get first error from errorTransfers 
     $this->assertSame('You have to enter a valid locale, example: en_US', $localeErrorTransfer->getMessage(), sprintf('Expected error message for invalid locale got "%s"', $localeErrorTransfer->getMessage()));
}
```

Reply:
I have do it like below please review

public function testAddManifestWithInvalidLocaleReturnsErrorResponse(): void
    {
        // Arrange
        $manifestRequestTransfer = $this->tester->haveManifestCreateRequest();
        $manifestRequestTransfer->getManifestOrFail()->setLocaleName(static::INVALID_LOCALE);

        // Act
        $manifestResponseTransfer = $this->tester->getFacade()->createManifest(
            $manifestRequestTransfer,
        );

        // Assert
        $this->assertCount(1, $manifestResponseTransfer->getErrors(), 
            sprintf(
                'Expected to get exactly "1" error, got "%s". Either there is no error or you have more than expected',
                $manifestResponseTransfer->getErrors()->count(),
            ),
        );
    }